astropy/units/tests/test_format.py

```diff
@@ -415,8 +397,7 @@ def test_roundtrip_dex(self, unit):
 
 
 class TestRoundtripOGIP(RoundtripBase):
-    format_ = "ogip"
-    deprecated_units = u_format.OGIP._deprecated_units | {"d"}
```

Comment:
I have no idea why `"d"` was added to the OGIP deprecated units. It is not needed for the tests here.

Reply:
This comes from a PR by me, #9005. I'm not sure why I added it, but surely it must have stopped a warning at the time... 

Note that inside `test_roundtrip` there is some special handling for "d" -- indeed, there's something weird about days:
```
 In [3]: u.Unit("day", format="ogip")
Out[3]: Unit("d")

In [4]: u.Unit("day", format="ogip").to_string("ogip")
WARNING: UnitsWarning: '24.0' scale should be a power of 10 in OGIP format [astropy.units.format.ogip]
Out[4]: '24 h'
```
But not for this PR!